opensearch-operator/pkg/builders/cluster_test.go

```diff
@@ -88,13 +88,13 @@ var _ = Describe("Builders", func() {
 				Value: "master",
 			}))
 		})
-		It("should use General.DefaultRepo for the InitHelper image if configured", func() {
-			var clusterObject = ClusterDescWithVersion("2.2.1")
-			customRepository := "mycustomrepo.cr"
-			clusterObject.Spec.General.DefaultRepo = &customRepository
-			var result = NewSTSForNodePool("foobar", &clusterObject, opsterv1.NodePool{}, "foobar", nil, nil, nil)
-			Expect(result.Spec.Template.Spec.InitContainers[0].Image).To(Equal("mycustomrepo.cr/busybox:1.27.2-buildx"))
-		})
+		//It("should use General.DefaultRepo for the InitHelper image if configured", func() {
```

Comment:
Instead of commenting out the test please modify it to test that if a custom repo is provided the inithelper image is still taken from the default repo.